src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveCourtVenueDetailsIntegrationTest.java

```diff
@@ -287,7 +287,7 @@ void shouldReturn200WhenParameterCourtTypeIdIsPassed() throws
                 path
             );
 
-        assertThat(response).isNotEmpty().hasSize(9);
+        assertThat(response).isNotEmpty().hasSize(10);
```

Comment:
add assertion for EXTERNAL_SHORT_NAME value

Reply:
done